CMSgov/design-system
# [NO-TICKET] Add a fill utility class for info status

## Summary

Add a fill utility class for info status for consistency. This is in support of https://github.com/CMSgov/design-system/pull/2416#discussion_r1134375955

## How to test

See that [the fill utility page](https://cmsgov.github.io/design-system/branch/pwolfert/add-info-fill-utility-class/utilities/background-color/) now has a section of info colors like [the theme colors page](https://cmsgov.github.io/design-system/branch/pwolfert/add-info-fill-utility-class/foundation/theme-colors/) does

## Checklist
